### PR TITLE
Convert `cursor.module.css` to CSS modules

### LINKED ISSUE #40521: Migrate global css in `frontend/src/metabase/css/core/cursor.module.css`

### PR DESCRIPTION
**Pull request by oisincoveney**
### Description

Describe the overall approach and the problem being solved.

### How to verify

Describe the steps to verify that the changes are working as expected.

1. New question -> Sample Dataset -> ...
2. ...

### Demo

_Upload a demo video or before/after screenshots if sensible or remove the section_

### Checklist

- [ ] Tests have been added/updated to cover changes in this PR

### CODE REVIEW

File: frontend/src/metabase/dashboard/components/ClickMappings.jsx [hunk 1]
```diff
@@ -267,7 +267,7 @@ function TargetWithSource({
           </div>
         </div>
         <div
-          className="cursor-pointer ml-auto"
+          className={cx(CS.cursorPointer, CS.mlAuto)}
```
[the remark this answers is not in the document]
Comment (oisincoveney) [reply frontend/src/metabase/dashboard/components/ClickMappings.jsx:270]: Ah I am getting caught doing all of these PRs. I'll stack them in the future

File: frontend/src/metabase/css/core/cursor.module.css [hunk 2]
```diff
@@ -1,15 +1,11 @@
-:global(.cursor-pointer),
-.cursor-pointer,
```
[the remark this answers is not in the document]
Comment (oisincoveney) [reply frontend/src/metabase/css/core/cursor.module.css:2]: Ah, didn't see the second one. I know the e2e test needs to be sorted but I'll do that in a sec

File: frontend/src/metabase/components/TokenField/TokenField.tsx [hunk 3]
```diff
@@ -648,7 +648,16 @@ class _TokenField extends Component<TokenFieldProps, TokenFieldState> {
...
+                  CS.py1,
+                  CS.pl1,
+                  CS.pr2,
+                  CS.block,
+                  CS.rounded,
+                  CS.textBold,
+                  CS.inlineBlock,
+                  CS.full,
+                  CS.cursorPointer,
+                  `text-${color}-hover`,
```
[the remark this answers is not in the document]
Comment (oisincoveney) [reply frontend/src/metabase/components/TokenField/TokenField.tsx:660]: Thankfully that's the only class name I can find that's dynamically generated. I've seen it a billion times and should be sorted with `colors.module`. 


And, of course as I look at it I didn't sort it in the colors module. I'll do that there

File: frontend/src/metabase/collections/components/MoveCollectionModal/MoveCollectionModal.tsx [hunk 4]
```diff
@@ -75,9 +75,9 @@ export const MoveCollectionModal = ({
   return (
     <MoveCollectionModalView
       collection={collection}
-      onMove={(source, destination) =>
-        dispatch(Collections.actions.setCollection(source, destination))
-      }
+      onMove={async (source, destination) => {
```
[the remark this answers is not in the document]
Comment (oisincoveney) [reply frontend/src/metabase/collections/components/MoveCollectionModal/MoveCollectionModal.tsx]: Oh what. Ok I'll reset to the last good commit and go from there

**Review by deniskaber — APPROVED** [review]

**Pull request merged**